README_fr.md

```diff
@@ -0,0 +1,1643 @@
+
+<a href="undo-sensitive-commit-push"></a>
+### J'ai accidentellement comitté et poussé des fichiers contenant des données sensibles
+
+If you accidentally pushed files containing sensitive data (passwords, keys, etc.), you can amend the previous commit. Keep in mind that once you have pushed a commit, you should consider any data it contains to be compromised. These steps can remove the sensitive data from your public repo or your local copy, but you **cannot** remove the sensitive data from other people's pulled copies. If you committed a password, **change it immediately**. If you committed a key, **re-generate it immediately**. Amending the pushed commit is not enough, since anyone could have pulled the original commit containing your sensitive data in the meantime. 
+
+Si vous avez accidentellement poussé des fichiers contenant des données sensibles (mots de passe, clés, etc.), vous pouvez modifier le commit précédent. Gardez toutesfois à l'esprit qu'une fois que vous avez poussé un commit, vous devez considérer n'importe quelle donnée qu'il contient comme étant compromise. Ces étapes peuvent supprimer les données sensibles de votre dépôt public ou de votre copie locale, mais vous ne pouvez **pas** supprimer les données sensibles des copies clônées par d'autres personnes. Si vous avez comitté un mot de passe, **changez-le immédiatement**. Si vous avez comitté une clé, **révoquez-la et régénérez-la immédiatement**. Modifier le commit poussé n'est pas suffisant, étant donné que n'importe qui aurait pu extraire le commit original contenant vos données sensibles pendant ce temps.
+
+Si vous souhaitez modifier le fichier et supprimer les données sensibles, lancez
+```sh
+(feature-branch)$ git add fichier_modifié
+(feature-branch)$ git commit --amend --no-edit
+(feature-branch)$ git push --force-with-lease origin [branche]
+```
+
+Si vous souhaitez supprimer tout un fichier (mais le garder en local), lancez
+```sh
+(feature-branch)$ git rm --cached fichier_sensible
+echo fichier_sensible >> .gitignore
+(feature-branch)$ git add .gitignore
+(feature-branch)$ git commit --amend --no-edit
+(feature-branch)$ git push --force-with-lease origin [branche]
+```
+Alternativement, stockez vos données sensibles dans des variables d'environnement locales.
+
+Si vous souhaitez supprimer tout un fichier (et ne pas le garder en local), lancez
+```sh
+(feature-branch)$ git rm fichier_sensible
+(feature-branch)$ git commit --amend --no-edit
+(feature-branch)$ git push --force-with-lease origin [branche]
+```
+
+Si vous avez créé d'autres commits pendant ce temps (en d'autres termes, les données sensibles sont dans un commit avant le commit précédent), vous devrez utiliser la fonction rebase.
```

Comment:
```suggestion
Si vous avez créé d'autres commits pendant ce temps (en d'autres termes, les données sensibles sont dans un commit avant le commit précédent), vous devrez utiliser la fonction `rebase`.
```
The Git methods names are usually highlighted using backticks "`" (for code formatting) in the rest of the document.